### PR TITLE
Stripping the ImageCategory class of its "Enum" derivative, creating a new AudioCategory class, and removing 2 troublesome test directories

### LINKED ISSUE #823: Add `AudioCategory` enum
## Description

We presently have an `ImageCategory` enum for the available image categories:

https://github.com/WordPress/openverse-catalog/blob/1506d6d9d391d7d12c8f5c515737623218036928/openverse_catalog/dags/common/loader/provider_details.py#L119-L122

It would be great if we could add a similar `AudioCategory` enum for our audio providers. This would include all of the available audio categories as defined by the API:

https://github.com/WordPress/openverse-api/blob/d9f83e53761dd502c93384f0aa6e2f7b711151e2/api/catalog/api/constants/field_values.py#L1-L8

## Additional context

It seems like every reference we have to one of the values is also `ImageCategories.<category>.value`, so I'm not sure we're getting much value out of this being an Python `Enum` derivative. We can probably just make this and the image categories an object that has each of those fields on it, e.g.:

```python
class ImageCategory:
    PHOTOGRAPH = "photograph"
    DIGITIZED_ARTWORK = "digitized_artwork"
    ILLUSTRATION = "illustration"
```

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by flamesjames**
## Fixes

Fixes #823  by @AetherUnbound 

## Description

This pull request solves 3 issues, 2 of which were described in #823:

1. Removing the "Enum" derivative from the existing `ImageCategory` class.
2. Adding a new `AudioCategory` class for the different Audio provider categories.
3. Removing to test log directories that were causing issues with cloning the forked project to local machine:

     - `tests/dags/common/test_resources/logs/test_dag/test_task/2021-08-10T00:00:00+0000/`
     - `tests/dags/common/test_resources/logs/test_dag/test_task/2021-08-20T00:00:00+0000/`

## Testing Instructions

For recreating the cloning issue, here are the steps I took (running on a Windows machine):

1. Forked the main project repo from WordPress.
2. Attempted to clone the forked repo to my local machine and got the following error:
`error: invalid path 'tests/dags/common/test_resources/logs/test_dag/test_task/2021-08-10T00:00:00+0000/1.log'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'`
3. Removing the 2 directories with the timestamp resolved the issue. 

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible
      errors.
     - I had issues with `just` and following the readme instructions for executing the tests so I was unable to perform this step.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by flamesjames** [comment]
Looking into these issues...

**Comment by flamesjames** [comment]
After 03bb96e46d6c4501935a083ff6d27fa5d453e922, lint issues were resolved but CI+CD Run Tests failed.. 

I'm seeing references to those 2 directories that I removed to fix a cloning issue with Windows (ex. lines 119, 144 of the failed test results log):
```
119: INFO     common.log_cleanup:log_cleanup.py:154 Found 1 log folders to delete: ['/opt/airflow/tests/dags/common/test_resources/logs/scheduler/2021-08-10']

144: [2023-01-24 20:30:43,721] {log_cleanup.py:154} INFO - Found 2 log folders to delete: ['/opt/airflow/tests/dags/common/test_resources/logs/scheduler/2021-08-20', '/opt/airflow/tests/dags/common/test_resources/logs/scheduler/2021-08-10']
```


I'm assuming that is the cause. Please let me know how I should move forward with this. Thank you!

**Comment by flamesjames** [comment]
@AetherUnbound any update on this? Let me know if I should make any modifications

**Comment by AetherUnbound** [comment]
Left a comment on the issue! We would like to keep those other files, can you restrict your PR changes to the files you edited only?

**Review by obulat — CHANGES_REQUESTED** [review]
The changes look great, @flamesjames!

Could you please restore the deleted log files?

**Comment by flamesjames** [comment]
@obulat @AetherUnbound the test log files have been added back and pushed with d717eaf1b014093c54be92dbaf890e8a3dfa229c

**Comment by AetherUnbound** [comment]
Pulling [this note](https://github.com/WordPress/openverse-catalog/issues/823#issuecomment-1410921416) forward from the issue - while the file renames are unrelated to this changeset they don't appear to negatively impact the existing tests _and_ they help Windows dev support, so I think they're OK to keep 🙂

File: openverse_catalog/dags/common/loader/provider_details.py [hunk 1]
```diff
@@ -118,32 +117,45 @@
 UA_STRING = f"Openverse/0.1 (https://wordpress.org/openverse; {CONTACT_EMAIL})"
 
 
-class ImageCategory(Enum):
+# Available Image Categories for API
+class ImageCategory:
+
```
Comment (AetherUnbound) [inline openverse_catalog/dags/common/loader/provider_details.py:122]:
```suggestion
```

File: openverse_catalog/dags/common/loader/provider_details.py [hunk 2]
```diff
@@ -118,32 +117,45 @@
...
     PHOTOGRAPH = "photograph"
     DIGITIZED_ARTWORK = "digitized_artwork"
     ILLUSTRATION = "illustration"
 
 
+# Available Audio Categories for API
+class AudioCategory:
+
```
Comment (AetherUnbound) [inline openverse_catalog/dags/common/loader/provider_details.py:130]:
```suggestion
```

**Review by AetherUnbound — APPROVED** [review]
This is fantastic, thank you and congrats on the contribution @flamesjames!

**Comment by flamesjames** [comment]
@AetherUnbound sorry about that - there you go!

**Review by obulat — APPROVED** [review]
Thank you for your contribution and taking time to get the code to work on Windows, @flamesjames !

**Comment by flamesjames** [comment]
@obulat @AetherUnbound happy to help! Thank you for the opportunity to contribute 😄

**Pull request merged**